test/molecule-role/molecule/_shared/helm/install-stackstate-agent.yml

```diff
@@ -7,11 +7,11 @@
     --set-string 'stackstate.cluster.name'='{{cluster_name}}' \
     --set-string 'stackstate.url'='http://{{ hostvars['kubernetes-cluster-agent']['ansible_host'] }}:7077/stsAgent' \
     --set "agent.image.tag={{agent_current_branch}}" \
-    --set "agent.image.repository=docker.io/stackstate/{{agent_docker_repo}}" \
+    --set "agent.image.repository=stackstate/{{agent_docker_repo}}" \
     --set "clusterAgent.image.tag={{agent_current_branch}}" \
-    --set "clusterAgent.image.repository=docker.io/stackstate/{{cluster_agent_docker_repo}}" \
+    --set "clusterAgent.image.repository=stackstate/{{cluster_agent_docker_repo}}" \
     --set "clusterChecks.image.tag={{agent_current_branch}}" \
-    --set "clusterChecks.image.repository=docker.io/stackstate/{{agent_docker_repo}}" \
+    --set "clusterChecks.image.repository=stackstate/{{agent_docker_repo}}" \
```

Comment:
Why does the repository variable no longer include docker.io?  Is it to be able to add it later on (perhaps substitute with quay.io)?

Reply:
I found that out lately as well, I think the helm charts now include pulling from docker.io